testapps/testapp/src/main/AndroidManifest.xml

```diff
@@ -34,6 +34,7 @@
     <uses-permission android:name="android.permission.USE_CREDENTIALS" />
 
     <application
+        android:networkSecurityConfig="@xml/network_security_config"
         android:name="com.microsoft.identity.client.testapp.MsalSampleApp"
         android:allowBackup="true"
```

Comment:
```suggestion
        android:allowBackup="false"
```

Reply:
This is unrelated to your changes, but there's no reason to set this value to true in our testapp 😄